pain-campagne.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: pain de campagne
+---
+
+# Le Pain de campagne
+
+![](img/le_pain/result.JPG)
+
+## Day one
+
+Prepare a _stiff starter:_
```

Comment:
`_stiff starter_:` non ? Quelle la règle typographique dans ce cas de figure?

Reply:
[](https://graphicdesign.stackexchange.com/questions/48540/in-a-sentence-ending-with-a-bold-or-italicised-word-does-the-period-need-to-be)

> all punctuation marks should be printed in the same style or font of type as the word or letter immediately preceding them.